soperator/installations/example/main.tf

```diff
@@ -96,9 +96,10 @@ module "nfs-server" {
   preset        = var.nfs.resource.preset
   instance_name = "${local.k8s_cluster_name}-nfs-server"
 
-  nfs_ip_range = data.nebius_vpc_v1_subnet.this.status.ipv4_private_cidrs[0]
-  nfs_size     = provider::units::from_gib(var.nfs.size_gibibytes)
-  nfs_path     = "/home"
+  nfs_disk_name_suffix = local.k8s_cluster_name
```

Comment:
This change is handled in-place without replacement by the Nebius Terraform Provider